StandardsRegExt.tex

```diff
@@ -1240,6 +1241,54 @@ \subsection{Defining Keys: StandardKey and StandardKeyURI}
 is not necessary to use the URI. 
 \end{admonition}
 
+\section{Operational Aspects}
+
+\subsection{Managment of Standards Records}
+
+Only IVOA standards that actually make use of identifiers to or into
+StandardsRegExt records MUST be accompanied with active records
+published by the ivoa.net registry.  This, in particular, concerns all
+standards defining capabilities to be discerned using their
+\xmlel{standardID} attribute.  Other standards MAY be registered in this
+way once they have entered the IVOA document repository.
+
+There are no regulations on StandardsRegExt records published under
+authorities other than ivoa.net; these can be published (and withdrawn)
+at any time.
+
+Editors of standards requiring an ivoa.net StandardsRegExt record SHOULD
+upload one when they submit the first Working Draft in order to make
+IVOIDs valid even in prototyping phases.  They MUST upload it at the
+beginning of the standard's RFC phase to ensure validators can work
+properly, and they MUST update the record when the standard reaches REC
+status.
+
+Submission to the ivoa.net registry generally happens through e-mail.
+See \url{http://rofr.ivoa.net/} for contact information.
```

Comment:
Can we be more specific ? Currently I don't see such a contact information on http://rofr.ivoa.net  other than that that page's support email address (support@canfar.net).